install/migrations/update_10.0.5_to_10.0.6/networkport.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ * @var array $ADDTODISPLAYPREF
+ */
+$ADDTODISPLAYPREF['NetworkPort'] = [3, 30, 31, 32, 33, 34, 35, 36, 38, 39, 40, 6];
```

Comment:
```suggestion
$ADDTODISPLAYPREF['NetworkPort'] = [6];
```

Reply:
This way add new entry to ```glpi_displaypreferences``` but with ```rank``` set to ```1```

Reply:
Before : 

```sql
MariaDB [10bugfixes_ruless]> select * from glpi_displaypreferences where itemtype = "NetworkPort";
+-----+-------------+-----+------+----------+
| id  | itemtype    | num | rank | users_id |
+-----+-------------+-----+------+----------+
| 266 | NetworkPort |   3 |    1 |        0 |
| 267 | NetworkPort |  30 |    2 |        0 |
| 268 | NetworkPort |  31 |    3 |        0 |
| 269 | NetworkPort |  32 |    4 |        0 |
| 270 | NetworkPort |  33 |    5 |        0 |
| 271 | NetworkPort |  34 |    6 |        0 |
| 272 | NetworkPort |  35 |    7 |        0 |
| 273 | NetworkPort |  36 |    8 |        0 |
| 274 | NetworkPort |  38 |    9 |        0 |
| 275 | NetworkPort |  39 |   10 |        0 |
| 276 | NetworkPort |  40 |   11 |        0 |
+-----+-------------+-----+------+----------+
11 rows in set (0.000 sec)
```

After 

```sql
MariaDB [10bugfixes_ruless]> select * from glpi_displaypreferences where itemtype = "NetworkPort";
+-----+-------------+-----+------+----------+
| id  | itemtype    | num | rank | users_id |
+-----+-------------+-----+------+----------+
| 266 | NetworkPort |   3 |    1 |        0 |
| 267 | NetworkPort |  30 |    2 |        0 |
| 268 | NetworkPort |  31 |    3 |        0 |
| 269 | NetworkPort |  32 |    4 |        0 |
| 270 | NetworkPort |  33 |    5 |        0 |
| 271 | NetworkPort |  34 |    6 |        0 |
| 272 | NetworkPort |  35 |    7 |        0 |
| 273 | NetworkPort |  36 |    8 |        0 |
| 274 | NetworkPort |  38 |    9 |        0 |
| 275 | NetworkPort |  39 |   10 |        0 |
| 276 | NetworkPort |  40 |   11 |        0 |
| 310 | NetworkPort |   6 |    1 |        0 |
+-----+-------------+-----+------+----------+
12 rows in set (0.001 sec)
```

Reply:
I already noticed this, indeed. We should probably handle it in migration class, globally, and in another PR.

Reply:
See [#0fc08ad](https://github.com/glpi-project/glpi/pull/13349/commits/0fc08ad82235d927a14e863e0c15380d5fbb4fa5), which used ```updateDisplayPrefs``` from ```MLigration``` class

Reply:
:+1: